avro.cabal

```diff
@@ -2,10 +2,10 @@
 --
 -- see: https://github.com/sol/hpack
 --
--- hash: fcff216c51da7f4f507714e81688ee3d9c149e101ffbe1b7051f4559eb2feebb
+-- hash: bfec925895943164224c08cad0c74436c45b1a75efd82d149d9643542ed7289d
 
 name:           avro
-version:        0.4.0.0
+version:        0.4.1.0
```

Comment:
Can you please not change the version in a PR? 
The reason is that there is an automated process of making a release when the version number changes in `master` and I would like to include some more changes when publishing the next release. So I'd like to merge your PR, but change the version number a bit later.

Reply:
Oops. Done, thanks!